felles/sikkerhet/src/main/java/no/nav/vedtak/sikkerhet/oidc/OidcProviderConfig.java

```diff
@@ -76,7 +75,9 @@ public static OidcProviderConfig instance() {
         idProviderConfigs.add(createOpenAmConfiguration(ENV.getProperty(OPEN_AM_WELL_KNOWN_URL)));
 
         // OIDC STS
-        idProviderConfigs.add(createStsConfiguration(ENV.getProperty(STS_WELL_KNOWN_URL)));
+        if (ENV.getProperty(STS_WELL_KNOWN_URL) != null || ENV.getProperty("oidc.sts.issuer.url") != null) { // Det er kanskje noen apper som ikke bruker STS token validering??
```

Comment:
Alle apper skal hente STS token i løpet av 2-3 måneder.
Hvem har ansvar for å sette STS_WELL_KNOWN_URL ?

Reply:
NAIS tar ikke ansvar for dette - så det er nok appene som må legge de inn i naiserator.yaml eller application*.properties

Reply:
Men || ENV.getProperty("oidc.sts.issuer.url") != null -> createStsConfiguration(ENV.getProperty(STS_WELL_KNOWN_URL) ?